modules/cnf-running-cnf-tests-in-discovery-mode.adoc

```diff
@@ -0,0 +1,274 @@
+// CNF-518 Running CNF tests in discovery mode
```

Comment:
Why not extending the current doc for cnf tests? 
I mean the one under https://docs.openshift.com/container-platform-ocp/4.5/scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.html#cnf-performing-end-to-end-tests-for-platform-verification_cnf-master 

Here there are infos that are duplicated from there, others are missing.